tests/streaming_tests.js

```diff
@@ -215,216 +113,207 @@ describe('streaming pipeline test', () => {
                 e.printSackTrace();
             }
 
-            streamSimple.flowInput = {
-                "flows": [
-                    {
-                        "name": "hkube_desc",
-                        "program": [
-                            {
-                                "rate": 150,
-                                "time": 140,
-                                "size": 80
-                            }
-                        ]
-                    },
-                    {
-                        "name": "hkube_desc",
-                        "program": [
-                            {
-                                "rate": 50,
-                                "time": 240,
-                                "size": 80
-                            }
-                        ]
-                    }
-                ],
-                "process_time": 0.02
-            }
+            streamSimple.flowInput = createFlowInput({
+                programs: [
+                    { rate: 150, time: 140 },
+                    { rate: 50, time: 240 }
+                ]
+            });
 
             const res = await runRaw(streamSimple);
             const { jobId } = res.body;
-            await waitForStatus(jobId, 'sen-1', 'active', 60000, 2000);
-            console.log("sen-1 is activce")
-            await delay(140);
-            await waitForStatus(jobId, 'sen-out-1', 'active', 60000, 2000);
-            console.log("sen-out-1 is activce")
-            await delay(5);
-            let required = await getRequiredPods(jobId, 'sen-1', 'sen-out-1');
+            await waitForStatus(jobId, statefulNodeName, 'active', 60 * 1000, 2 * 1000);
```

Comment:
To be more aligned with the time of the statefull sender rates. better to put the delay after the statefull is active. Then you don't wait both full waiting time for statelss plus delay.